### PR TITLE
Initial RC 2019-02-12

### PR DESCRIPTION
**Pull request by vladimir-shcherbakov**
Initial RC 2019-02-12

## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by vladimir-shcherbakov** [comment]
signing: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign-esrp/136/

File: src/Resources/Resources/ChangeLog.md [hunk 1]
```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+
+## Version 1.1.2
 * Fix for issue https://github.com/Azure/azure-powershell/issues/8166
```
Comment (markcowl) [inline src/Resources/Resources/ChangeLog.md]: This is ok for right now, but we should update this description to include a short summary of the issue fixed along with the link.

File: src/Resources/Resources/Az.Resources.psd1 [hunk 2]
```diff
@@ -169,13 +169,8 @@ PrivateData = @{
...
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Fix incorrect examples in ''New-AzADAppCredential'' and ''New-AzADSpCredential'' reference documentation
-* Fix issue where path for ''-TemplateFile'' parameter was not being resolved before executing resource group deployment cmdlets
-* Az.Resources: Correct documentation for New-AzureRmPolicyDefinition -Mode default value
-* Az.Resources: Fix for issue https://github.com/Azure/azure-powershell/issues/7522
-* Az.Resources: Fix for issue https://github.com/Azure/azure-powershell/issues/5747
-* Fix formatting issue with ''PSResourceGroupDeployment'' object
-    - More information here: https://github.com/Azure/azure-powershell/issues/2123'
+        ReleaseNotes = '* Fix for issue https://github.com/Azure/azure-powershell/issues/8166
```
Comment (markcowl) [inline src/Resources/Resources/Az.Resources.psd1]: See below - we should always include a description with the link

File: ChangeLog.md [hunk 3]
```diff
@@ -1,3 +1,27 @@
...
+#### Az.Compute
+* AEM extension: Add support for UltraSSD and P60,P70 and P80 disks
+* Update help description for Set-AzVMBootDiagnostics
+* Update help description and example for Update-AzImage
+
+#### Az.RecoveryServices
+General availability for Az.RecoveryServices module.
+
+#### Az.Resources
+* Fix for issue https://github.com/Azure/azure-powershell/issues/8166
```
Comment (markcowl) [inline ChangeLog.md]: see comments below - we should include a short (one sentence0 summary for each fixed bug.

**Comment by vladimir-shcherbakov** [comment]
signing: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign-esrp/140/

**Comment by markcowl** [comment]
signing run: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign-esrp/143/

**Pull request merged**